layers.go

```diff
@@ -677,7 +691,7 @@ func (r *layerStore) Unmount(id string) error {
 		layer.MountPoint = ""
 		err = r.Save()
 	}
-	return err
+	return err != nil, err
```

Comment:
If the `Put()` succeeds but we fail to save the updated mount count, we return `true` here.  Is that the intention?

Reply:
you tell me.  The storage has unmounted the file system, but the database does not have the correct data.  Does this mean it is mounted or not?